src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
```

Comment:
Check this place, if you want. I think here you can add some padding, because for me it looks tight :D 
![image](https://github.com/mate-academy/layout_landing-page/assets/142230575/57a077fd-249e-421f-8f43-a67635a4bf62)